repository/src/main/java/com/epam/esm/certificate/model/Certificate.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.esm.certificate.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.*;
+
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
```

Comment:
I am not sure if we can use Lombok in this task

Reply:
Mentee can use lombok when agreed with mentor.